src/AutoRest.CSharp.V3/Generation/Writers/JsonCodeWriterExtensions.cs

```diff
@@ -277,7 +277,10 @@ public static void ToSerializeCall(this CodeWriter writer, JsonSerialization ser
                                     writer.Append($"continue;");
                                 }
                             }
-                            else if (!property.IsRequired)
+                            else if (!property.IsRequired &&
```

Comment:
This is the change. So you don't have to scroll :)

Reply:
Thank you! :)